server/lib/config/pbench-server-default.cfg

```diff
@@ -68,9 +68,6 @@ workers = 3
 # Set the gunicorn worker timeout. Setting it to 0 has the effect of infinite timeouts
 worker_timeout = 9000
 
-# Default roles this pbench server takes on, see crontab roles below.
-roles = pbench-results
-
```

Comment:
Ditto for `server/pbenchinacan/etc/pbench-server/pbench-server.cfg`?

Reply:
No: we only define the crontab roles in the *default* config file. We don't duplicate that in `pbench-server.cfg`.

Reply:
OK.  What does [this line](https://github.com/dbutenhof/pbench/blob/f5f70532ea65cbd7c24e8b2821415691c1cf43e9/server/pbenchinacan/etc/pbench-server/pbench-server.cfg#L14) do?

Reply:
It looks like it's doing the backstroke ... but it's in the wrong pond. 😦